manager-rs/dauth-service/Cargo.toml

```diff
@@ -19,6 +19,8 @@ serde_yaml = "0.8"
 structopt = "0.3"
 rand = "0.7"
 ed25519-dalek = "1.0"
+tempfile = "3.3"
```

Comment:
It's a minor nit, but since you asked for rust-specific feedback, since you're using tempfile just for running tests, you probably want it to be a dev dependency (https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies), so it doesn't have to be pulled and built for a general build.

Reply:
This is a really good thing to know, thanks!

Reply:
+1 : )